ttnn/CMakeLists.txt

```diff
@@ -61,23 +68,33 @@ target_precompile_headers(ttnn_lib PRIVATE
     <vector>
 )
 
-set(TTNN_FINAL_SRC $<TARGET_OBJECTS:ttnn_lib>)
-set(TTNN_PUBLIC_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/cpp)
-set(TTNN_PUBLIC_LINK_LIBS compiler_flags linker_flags tt_eager) # linker_flags = -rdynamic if tracy enabled
+### Setup TTNN as a shared library with optional Python bindings
+set(TTNN_FINAL_SRC $<TARGET_OBJECTS:ttnn_lib> ${QUEUE_SRCS} ${TENSOR_SRCS} ${TT_DNN_SRCS})# $<TARGET_OBJECTS:tt_dnn> $<TARGET_OBJECTS:tensor> $<TARGET_OBJECTS:queue>)
```

Comment:
Do we need to keep this comment appended here?

Reply:
Thanks!